src/app/components/NeedForm/NeedForm.tsx

```diff
@@ -19,12 +17,10 @@ export default function NeedForm({
 
   function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
     event.preventDefault()
-    const newNeed = {
-      id: nanoid(),
+    const payload = {
```

Comment:
Why did you change this to 'payload'? 'Newneed' would be more understandable, I think.